Notion Release Notes Agent/polling_server.py

```
@@ -0,0 +1,346 @@
+                parameters={
+                    "owner": Settings.GITHUB_REPO_OWNER,
+                    "repo": Settings.GITHUB_REPO_NAME,
+                    "state": "closed",  # Get closed PRs
+                    "per_page": 30,  # Check last 30 closed PRs
+                    "sort": "updated",
+                    "direction": "desc"
+                },
+            )
+
+            if not result:
+                logger.error(f"❌ Failed to fetch PRs")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)